recipes/vtk-cdat/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+python:
+    - 3.7
+    - 2.7
```

Comment:
You shouldn't include python here

Reply:
@scopatz  For now, I want to exclude python 3.6 build so that I do not exceed the 6 hours limit build time, is that possible?

Reply:
Oh yeah, use the following

```yaml
build:
  skip: true  # [py == 36]
```